sources/procfs.go

```diff
@@ -675,6 +675,31 @@ func parseFileElements(path string, directoryDepth int) (name string, nodeName s
 	return name, nodeName, nil
 }
 
+func convertToBytes(s string) string {
+	new_s := ""
+	multiplier := 1
+	switch finalChar := s[len(s)-1:]; finalChar {
+	case "K":
+		new_s = strings.TrimSuffix(s, "K")
+		multiplier = 1000
+	case "M":
+		new_s = strings.TrimSuffix(s, "M")
+		multiplier = 1000000
+	case "G":
+		new_s = strings.TrimSuffix(s, "G")
+		multiplier = 1000000000
+	default:
+		//this should only happen for integers
+		return s
+	}
+	byteVal, err := strconv.Atoi(new_s)
+	if err != nil {
+		return s
+	}
+	byteVal = byteVal * multiplier
+	return strconv.Itoa(byteVal)
```

Comment:
No worries my friend! Hmm... any reason for float64 as opposed to just int64 (or... uint64)? I like that idea though, just force it all to that value just to be safe. Heck, what if there's some bug where Lustre has the metrics as bytes in the code but it's actually in the GB size, then we will still run into this issue. It's a stretch... but I'd rather be safe. Regardless of that though, I like the idea of forcing float64(/int64/uint64) on everything.